GPy/inference/latent_function_inference/expectation_propagation.py

```diff
@@ -81,35 +82,40 @@ def __init__(self, mu, Sigma, L=None):
         Sigma_diag = np.diag(self.Sigma)
         super(posteriorParams, self).__init__(mu, Sigma_diag)
 
-    def _update_rank1(self, delta_tau, ga_approx, i):
-        ci = delta_tau/(1.+ delta_tau*self.Sigma_diag[i])
-        DSYR(self.Sigma, self.Sigma[:,i].copy(), -ci)
-        self.mu = np.dot(self.Sigma, ga_approx.v)
+    def _update_rank1(self, mean_prior, delta_tau, delta_v, ga_approx, i):
```

Comment:
Does mean_prior need to be passed in if it is never used, is its effect somehow ? Maybe this is more general for other algorithms that are based off this posterior?